src/rules/noShadowedVariableRule.ts

```diff
@@ -158,6 +153,15 @@ class NoShadowedVariableWalker extends Lint.BlockScopeAwareRuleWalker<ScopeInfo,
     }
 }
 
-class ScopeInfo {
-    public variableNames: string[] = [];
+// ES6 Set shim for strings
+class Set {
```

Comment:
Do we really need to shim ES6 features? Maybe we should just specify the correct libs for tsc